docs/source/release/v4.1.0/reflectometry.rst

```diff
@@ -2,6 +2,9 @@
 Reflectometry Changes
```

Comment:
I don't really think the copyright date is significant, regarding the test files, yes you need a lot to do a meaningful reduciton, what I had to do is to copy the same files from system test data also to unit and doc test data, since apparently the doc tests and usage examples are run without system test data.

Reply:
Are all of the test files needed in each of those locations? It looks like only a small subset are used in the unit tests? And it looks like some (e.g. 397805.nxs) are not used at all? Unless I'm missing something!

Reply:
Regarding the release notes, it looks like the content has been moved to v4.3 but there are some formatting changes remaining in v4.1. Could you revert those just to keep things tidy? I find it confusing to see changes to old release notes!

Reply:
I think the copy of files in docs tests is probably not needed, I will try to remove them. It's true, not all of the files are used in the current tests, I thought about writing more tests later, when it's more stabilized, but prepare the data already, but maybe I should indeed remove those for the moment. As for the release notes, there was some title level mismatch in 4.1 that I had to fix to get the build passing without warnings.